src/Orleans/Streams/Predicates/ExactMatchStreamNamespacePredicate.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace Orleans.Streams
+{
+    [Serializable]
+    internal class ExactMatchStreamNamespacePredicate : IStreamNamespacePredicate
```

Comment:
Yes I think you are correct! Although I'm not sure it is the correct name. 
RegexStreamNameSpacePredict didn't change the nature of ImplicitSubscribe, which is subscribe through annotation on grain class. It just change the underlying mechanism of name space filtering. 
But I cannot think of a better name. I guess I'm fine with the name. If you can come up with a better one then it is great!